uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java

```diff
@@ -0,0 +1,99 @@
+package com.microsoft.identity.client.ui.automation.app;
+
+import androidx.annotation.NonNull;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiObjectNotFoundException;
+
+import com.microsoft.identity.client.ui.automation.installer.LocalApkInstaller;
+import com.microsoft.identity.client.ui.automation.interaction.PromptHandlerParameters;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.MicrosoftStsPromptHandler;
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.MicrosoftStsPromptHandlerParameters;
+import com.microsoft.identity.client.ui.automation.logging.Logger;
+import com.microsoft.identity.client.ui.automation.utils.UiAutomatorUtils;
+import com.microsoft.identity.common.java.exception.ServiceException;
+import com.microsoft.identity.common.java.providers.oauth2.IDToken;
+
+import org.junit.Assert;
+
+import java.util.Map;
+
+/**
+ * A model for interacting with the Msal Test App for MSAL Android during UI Test.
+ */
+
+public class MsalTestApp extends App{
+
+    private final static String TAG = MsalTestApp.class.getSimpleName();
+    private static final String MSAL_TEST_APP_PACKAGE_NAME = "com.microsoft.identity.client.testapp";
+    private static final String MSAL_TEST_APP_NAME = "MSAL Test App";
+    public static final String MSAL_TEST_APP_APK = "MsalTestApp.apk";
+    public static final String OLD_MSAL_TEST_APP_APK = "OldMsalTestApp.apk";
+
+
+
+    // constructors
+    public MsalTestApp() {
+        super(MSAL_TEST_APP_PACKAGE_NAME, MSAL_TEST_APP_NAME, new LocalApkInstaller());
+        localApkFileName = MSAL_TEST_APP_APK;
+    }
+
+    public MsalTestApp(@NonNull final String msalTestAppApk, @NonNull final String updateMsalTestAppApk) {
```

Comment:
The params are the APKs themselves or just name of the apk file? If the latter then please rename params appropriately.

Reply:
Just name of the apk file, MsalTestApp.apk and OldMsalTestApp.apk separately. Should I change the name to these two names?